lib/web/apiserver.go

```diff
@@ -2566,10 +2612,9 @@ func (h *Handler) hostCredentials(w http.ResponseWriter, r *http.Request, p http
 //
 // { "user": "bob", "password": "pass", "otp_token": "tok", "pub_key": "key to sign", "ttl": 1000000000 }
 //
-// Success response
+// # Success response
```

Comment:
Not sure why but the Go extension keeps autoformatting it like this on every save, can't figure out how to turn that off.